nautobot/extras/jobs.py

```diff
@@ -385,53 +382,29 @@ def file_path(cls):  # pylint: disable=no-self-argument
     @classproperty
     def class_path(cls):  # pylint: disable=no-self-argument
         """
-        Unique identifier of a specific Job class, in the form <source_grouping>/<module_name>/<ClassName>.
+        Unique identifier of a specific Job class, in the form <module_name>.<ClassName>.
 
         Examples:
-        local/my_script/MyScript
-        system/nautobot.core.jobs/MySystemJob
-        plugins/my_plugin.jobs/MyPluginJob
-        git.my-repository/myjob/MyJob
+        my_script.MyScript
+        nautobot.core.jobs.MySystemJob
+        my_plugin.jobs.MyPluginJob
+        git_repository.jobs.myjob.MyJob
         """
-        # TODO(Glenn): it'd be nice if this were derived more automatically instead of needing this logic
-        # TODO(jathan: Once we make all Jobs imported as modules, this may no longer be necessary as
-        # we can just trust that the import location and class_path can be derived from a single
-        # source.
-        if cls in registry["plugin_jobs"]:
-            source_grouping = "plugins"
-        elif cls in registry["system_jobs"]:
-            source_grouping = "system"
-        elif cls.file_path.startswith(settings.JOBS_ROOT):
-            source_grouping = "local"
-        elif cls.file_path.startswith(settings.GIT_ROOT):
-            # $GIT_ROOT/<repo_slug>/jobs/job.py -> <repo_slug>
-            source_grouping = ".".join(
-                [
-                    "git",
-                    os.path.basename(os.path.dirname(os.path.dirname(cls.file_path))),
-                ]
-            )
-        else:
-            raise RuntimeError(
-                f"Unknown/unexpected job file_path {cls.file_path}, should be one of "
-                + ", ".join([settings.JOBS_ROOT, settings.GIT_ROOT])
-            )
-
-        return "/".join([source_grouping, cls.__module__, cls.__name__])
+        return f"{cls.__module__}.{cls.__name__}"
 
     @classproperty
     def class_path_dotted(cls):  # pylint: disable=no-self-argument
         """
         Dotted class_path, suitable for use in things like Python logger names.
```

Comment:
Should this be documented as deprecated in favor of `class_path`?

Reply:
Good thought, yes.